pages/linux/eix.md

```diff
@@ -0,0 +1,27 @@
+# eix
+
+> Utilities for searching local Gentoo packages.
+
+- Synchronize local cache (should be ran after installation/updating packages):
+
+`eix-update`
+
+- Search for a package:
+
+`eix {{package_name}}`
+
+- Search for installed packages:
+
+`eix -I {{package_name}}`
```

Comment:
Is there a long form of this argument? We generally prefer long forms in tldr pages - same for the other examples below.